api/RestfulService.php

```diff
@@ -67,6 +67,22 @@ public static function set_default_curl_options($optionArray) {
 	}
 
 	/**
+	 * Parses cURL options defined in YML configuration.
+	 *
+	 * @return array
+	 */
+	public static function get_default_curl_options() {
+		$rawOptions = Config::inst()->get('RestfulService', 'default_curl_options');
```

Comment:
Actually I like too. Simpler to read and right as well as providing the same exact benefits and separation of concerns. 


- Patrick Nelson
Sent from my phone.

> On Apr 22, 2015, at 5:01 PM, Damian Mooyman <notifications@github.com> wrote:
> 
> In api/RestfulService.php:
> 
> > @@ -67,6 +67,22 @@ public static function set_default_curl_options($optionArray) {
> >  	}
> >  
> >  	/**
> > +	 * Parses cURL options defined in YML configuration.
> > +	 *
> > +	 * @return array
> > +	 */
> > +	public static function get_default_curl_options() {
> > +		$rawOptions = Config::inst()->get('RestfulService', 'default_curl_options');
> I personally prefer static::config()->default_curl_options but you can use get_called_class if you like.
> 
> —
> Reply to this email directly or view it on GitHub.
>